tools/const_finder/test/const_finder_test.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert' show jsonEncode;
+import 'dart:io';
+
+import 'package:const_finder/const_finder.dart';
+import 'package:path/path.dart' as path;
+
+int exitCode = 0;
```

Comment:
fyi this will shadow the top-level called `exitCode` from `dart:io`.

Reply:
TIL